tw-tasks-core/src/main/java/com/transferwise/tasks/health/ClusterWideTasksStateMonitor.java

```diff
@@ -294,4 +333,9 @@ public Integer getStuckTasksCount() {
   public List<Pair<String, Integer>> getErroneousTasksCountPerType() {
     return erroneousTasksCountPerType;
   }
+
+  @Override
+  public List<Pair<String, Integer>> getStuckTasksCountPerType() {
+    return stuckTasksCountPerType;
```

Comment:
I guess the "right" way to expose this data would be make a copy under lock, like

```java
public List<Pair<String, Integer>> getStuckTasksCountPerType() {
  stateLock.lock();
  try {
    return ImmutableList.copyOf(stuckTasksCountPerType);
  } finally {
    stateLock.unlock();
  }
}
```

Reply:
It is actually safe to do it this way, created map is basically immutable.

The lock in current form can create lots of lag.